sdk/core/core-http/lib/coreHttp.ts

```diff
@@ -1,7 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-export { WebResource, HttpRequestBody, RequestPrepareOptions, HttpMethods, ParameterValue, RequestOptionsBase, TransferProgressEvent } from "./webResource";
+export {
+  WebResource,
+  HttpRequestBody,
+  RequestPrepareOptions,
+  HttpMethods,
+  ParameterValue,
+  RequestOptionsBase,
+  TransferProgressEvent
+} from "./webResource";
```

Comment:
Now that #4863 is merged to master, please update the branch for this PR from the latest changes in master to reduce the noise